src/site/content/en/blog/api-for-fast-beautiful-web-fonts/index.md

```diff
@@ -0,0 +1,162 @@
+# Find the official list of tags at https://stackoverflow.com/tags.
+# stack_overflow_tag: service-worker
+---
+
+Over the years, a lot has changed with <a href="https://fonts.google.com/knowledge/glossary/web_font">web fonts</a> technology. What used to be a niche practice that required an image of text or a <a href="https://en.wikipedia.org/wiki/Scalable_Inman_Flash_Replacement">Flash</a> plug-in (and that compromised your website’s search engine optimization) is now a standard practice across the web. Once upon a time, you had to load an entire font before the page loaded—with styles and characters you may not have even used—but even that has become a thing of the past.
+
+The Google Fonts <a href="https://developers.google.com/fonts/docs/getting_started">CSS API</a> has also evolved over the years to keep up with changes in web fonts technology. It has come a long way from its original value proposition—to make the web faster by allowing your browser to cache commonly used fonts across all the websites that used the API. This is <a href="https://developer.chrome.com/blog/http-cache-partitioning/#how-will-cache-partitioning-affect-chromes-http-cache">no longer true</a>, but the API still provides additional and important optimizations so that websites load quickly and the fonts work well. 
+
+Using the Google Fonts CSS API, your website can request only the font data it needs to keep its CSS loading time to a minimum, ensuring your website visitors can load your content as quickly as possible. The API will respond to each request with the best font for that web browser.
+
+All this happens by including a single line of HTML in your code.
+
+## How to use the Google Fonts CSS API
+
+The <a href="https://developers.google.com/fonts/docs/getting_started">Google Fonts CSS API documentation</a> sums it up nicely:
+
+<blockquote>
+  <p>
+    You don't need to do any programming; all you have to do is add a special stylesheet link to your HTML document, then refer to the font in a CSS style.
+  </p>
+  <cite></cite>
+</blockquote>
+
+The minimum you need to do is include a single line in your HTML, like this:
+
+```html
+<link href="https://fonts.googleapis.com/css2?family=Roboto+Mono&display=swap" rel="stylesheet" />
+```
+
+When you request a font from the API, you specify which family or families you would like and, optionally, their weights, styles, subsets, and <a href="https://developers.google.com/fonts/docs/getting_started">many other options</a>. The API will then handle your request in one of two ways:
+
+1. If your request uses common parameters that the API already has files for, it will immediately return CSS to your user, directing them to these files.
+2. If you requested a font with parameters that the API doesn’t currently have cached, it will subset your fonts on the fly, using <a href="https://harfbuzz.github.io/harfbuzz-hb-subset.html">HarfBuzz</a> to do it quickly, and return CSS pointing to them.
```

Comment:
```suggestion
2. If you requested a font with parameters that the API doesn’t currently have cached, it will subset your fonts on the fly, using [HarfBuzz](https://harfbuzz.github.io/harfbuzz-hb-subset.html) to do it quickly, and return CSS pointing to them.
```